docs/cookbook/deployment/platform-sh.rst

```diff
@@ -93,15 +93,15 @@ This is how this file should look like for Sylius (tuned version of the default
 
     mounts:
         "/var/cache": "shared:files/cache"
-        "/var/logs": "shared:files/logs"
-        "/web/uploads": "shared:files/uploads"
-        "/web/media": "shared:files/media"
+        "/var/log": "shared:files/logs"
+        "/public/uploads": "shared:files/uploads"
+        "/public/media": "shared:files/media"
 
     hooks:
         build: |
-            rm web/app_dev.php
-            rm web/app_test.php
-            rm web/app_test_cached.php
+            rm public/app_dev.php
```

Comment:
These three lines are not needed since there is only `public/index.php` in Sylius v1.3

Reply:
I think I will change this one again in a speparate PR, whenever we learn how should the platform.sh config look like on SyliusDemo 😄

Reply:
This one I'm 100% sure about though :)